config/dev.exs

```diff
@@ -14,8 +14,10 @@ config :phoenix, :plug_init_mode, :runtime
 # hostname(string) - (for Static) provides a constant IP address for Static (ex. "127.0.0.1")
 # port(pos_int) - (for Static) provides a P2P port number (ex. 3002)
 #
-# config :uniris, Uniris.Networking, 
-#   port: 4000
+config :uniris, Uniris.Networking, 
+  # ip_provider: Uniris.Networking.IPLookup.Static,
+  # hostname: "127.0.0.1",
+  port: 5454
```

Comment:
Why did you change the default port ?

Reply:
just tested. Config is reverted to 3002 now